tests/test_maybe/test_maybe_base.py

```diff
@@ -1,7 +1,7 @@
-from returns.maybe import Maybe
+from returns.maybe import Some, _Nothing
```

Comment:
This test can be deleted, the only purpose was to ensure we put a type inside `Maybe.success_type` and `Maybe.failure_type`